doc/whats_new/v0.24.rst

```diff
@@ -536,6 +536,10 @@ Changelog
   will raise a `ValueError` when fitting on data with all constant features.
   :pr:`18370` by :user:`Trevor Waite <trewaite>`.
 
+- |Fix| Using `sort_results=True` now correctly sorts the results in methods
+  `radius_neighbors` and `radius_neighbors_graph`, even when using the "brute"
+  algorithm. :pr:`18612` by `Tom Dupre la Tour`_.
```

Comment:
Should we mention the affected classes, or are there too many of them?

Reply:
Done